test/starknet/vanilla_space.ts

```diff
@@ -48,31 +49,53 @@ async function setup() {
   };
 }
 
+function setupProposeCalldata(
```

Comment:
This function is useless, it only take argument to return it in slightly different way, you could do that directly on line 84

Reply:
[Add sx-api dep; Remove setupcalldata function](https://github.com/snapshot-labs/sx-core/pull/54/commits/fecb1fc3d7d126c81e82b9063706002c758b0230)